qemu/tests/qemu_guest_agent.py

```diff
@@ -885,6 +893,100 @@ def gagent_check_fsfreeze(self, test, params, env):
         # Finally, do something after thaw.
         self._action_after_fsthaw(test, params, env)
 
+    @error_context.context_aware
+    def gagent_check_hotplug_virtio_console(self, test, params, env):
```

Comment:
It's better to move gagent_check_hotplug_virtio_console function out to be a single py file,such as qemu_guest_agent_snapshot.py  and qemu_guest_agent_suspend.py,which will be more clear.

Reply:
After I directly use chardev hotplug method in avocado-vt, the code will be reduced. And if I put it in current py file, the code logical is same as other cases and clear. I think it's ok.